lib/jpegli/huffman.h

```diff
@@ -21,10 +21,6 @@ constexpr int kJpegHuffmanRootTableBits = 8;
 constexpr int kJpegHuffmanLutSize = 758;
 
 struct HuffmanTableEntry {
-  // Initialize the value to an invalid symbol so that we can recognize it
-  // when reading the bit stream using a Huffman code with space > 0.
-  HuffmanTableEntry() : bits(0), value(0xffff) {}
```

Comment:
Please set this value after the zero-initialization in decode.cc (e.g.  with simple for loop over all kAllHufLutSize entries)

Reply:
done (if I understood correctly..)
I guess the `bits` don't have to be set to zero again